examples/iterate_hist_forecast.py

```diff
@@ -53,11 +54,32 @@ async def main(service_address: str) -> None:
         features=features, locations=locations, start=start, end=end
     )
 
-    async for location_forecast in location_forecast_iterator:
-        for forecasts in location_forecast.forecasts:
-            print("Timestamp:", forecasts.valid_at_ts)
-            for feature_forecast in forecasts.features:
-                print(feature_forecast)
+    arrays = []
+    async for forecasts in location_forecast_iterator:
+        # You can work directly with the protobuf object forecasts.
+        # Here we choose to flatten into into a numpy array instead.
+        arr = forecasts.flatten()
+        arrays.append(arr)
+    data = np.concatenate(arrays)
+
+    # Optionally, you can construct a pandas dataframe from the data.
+    # pylint: disable=import-outside-toplevel, import-error
+    import pandas as pd  # type: ignore[import]
```

Comment:
We have a policy not to use pandas in production for now. But this is just an example and we may use it in our experiments.

Reply:
Also pylint ignores could not be inline since flake would complain about the line being too long.